src/sentry/workflow_engine/migration_helpers/issue_alert_conditions.py

```diff
@@ -356,15 +359,16 @@ def create_event_unique_user_frequency_condition_with_conditions(
     if conditions:
         for condition in conditions:
             if condition["id"] in (EventAttributeFilter.id, TaggedEventFilter.id):
+                match = MatchType(condition["match"])
                 comparison_filter = {
-                    "match": condition["match"],
+                    "match": match,
                     "key": (
                         condition["attribute"]
                         if condition["id"] == EventAttributeFilter.id
                         else condition["key"]
                     ),
                 }
-                if comparison_filter["match"] not in {MatchType.IS_SET, MatchType.NOT_SET}:
+                if match not in {MatchType.IS_SET, MatchType.NOT_SET}:
```

Comment:
do we need to do anything if it is a match type?

Reply:
all match types should have `key`, and only non-is_set/not_set match types should contain the `value`